packages/expo-cli/src/commands/client/tagger.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @flow
+ */
+
+class Updater {
+  constructor(updateAllFn) {
+    this.updateAllFn = updateAllFn;
+  }
+
+  async updateAllAsync(objs: Array<Object>) {
+    const taggedObjs = objs.filter(obj => obj._shouldUpdate);
+    clearTags(objs);
+    return await this.updateAllFn(taggedObjs);
+  }
+}
+
+function tagForUpdate(obj: Object) {
+  obj._shouldUpdate = true;
```

Comment:
It might be better to use a Symbol for that (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) but generally doesn't matter.